docs/whats-new.asciidoc

```diff
@@ -13,114 +13,141 @@ Other versions: {security-guide-all}/8.13/whats-new.html[8.13] | {security-guide
 
 [float]
-=== Access the entity details flyout from the Entity Analytics dashboard 
+=== Use AI Assistant when writing rule queries
 
-Clicking on a specific host or user name in the {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] now opens the host or user details flyout instead of the host or user details page. This allows you to access entity metadata and risk score information without navigating away from the dashboard.
+When creating rules, use AI Assistant to improve rule queries or to quickly correct them.
 
 [float]
-=== Entity details flyout shows contribution scores per alert
+=== Bulk update custom highlighted fields for rules
 
-The **Risk contributions** section of the {security-guide}/hosts-overview.html#host-details-flyout[entity details flyout] now shows the top 10 alerts that contributed to the latest risk scoring calculation and each alert's contribution score. This makes each entity's risk score easier to understand and gives better insight into which alerts you should investigate at the entity level.
+Bulk add or remove {security-guide}/rules-ui-create.html#rule-ui-advanced-params[custom highlighted fields] for multiple detection rules.
 
-[role="screenshot"]
-image::whats-new/images/8.14/contribution-scores-per-alert.png[Contribution scores for top 10 alerts, 90%]
+[float]
+=== Preview entities and alerts in the alert details flyout
+
+You can now preview host and user details from the **Insights** tab of the {security-guide}/view-alert-details.html[alert details flyout] instead of going to the **Hosts** or **Users** pages for more information. From the **Correlations** tab in the flyout, you can also preview alerts that are related to each other instead of leaving the flyout to access them.
 
 [float]
-== Detection rules and alerts enhancements
+=== Expandable alert details flyout enabled by default
 
+The expandable alert details flyout is now enabled by default in multiple places throughout the {security-app}.
 
 [float]
-=== Value list improvements
+==  Improvements to the Timeline data exploration experience 
+
+Several improvements have been made to enhance your data exploration experience in Timeline: 
 
-You can now {security-guide}/value-lists-exceptions.html#edit-value-lists[edit value lists] from the UI, wherever you use them. For example, you can now add items to a value list while creating a rule exception that references that value list.
+- Multiple components from Discover have been incorporated, such as the sidebar and table, which allow you to quickly find fields of interest. 
+- You can now toggle row renderers, which allow you to easily add or remove remove context from events.
+- Notes are easier to add and track from the new Notes flyout.
 
 [role="screenshot"]
-image::whats-new/images/8.14/edit-value-lists.png[Edit items in a value list, 90%]
+image::whats-new/images/8.15/timeline-ui-renderer.png[Example Timeline with the event renderer highlighted]
```

Comment:
Here are additional images in case you wanted to show each change:

#### Sidebar and table
<img width="1734" alt="timeline-sidebar" src="https://github.com/user-attachments/assets/d1d2b1d1-c697-4cb0-b7f2-762d11896e90">


#### Notes flyout
<img width="2165" alt="timeline-notes-flyout" src="https://github.com/user-attachments/assets/a240d13e-ff6d-48fa-a984-82ed1c4f46bf">